lib/graphql/schema.ts

```diff
@@ -1055,6 +1054,14 @@ export type TransactionsConnectionEdge = {
   cursor: Scalars['String'],
 };
 
+export type TransactionSplit = {
+   __typename?: 'TransactionSplit',
+  id: Scalars['ID'],
+  amount: Scalars['Int'],
+  category: TransactionCategory,
+  userSelectedBookingDate?: Maybe<Scalars['DateTime']>,
```

Comment:
Should this be mandatory? And we pass `null` when there is no booking date provided?
So like this
```
userSelectedBookingDate: Maybe<Scalars['DateTime']>,`
```
I thought that's what the `Maybe` meant. 
So basically I'm saying, can we/should we remove the question mark?

Reply:
`export type Maybe<T> = T | null;` as defined in the top of the file
when we create a split, userSelectedBookingDate is optional, Maybe allowed 'null' but '?' allowed 'undefined', so it can either be `null`, 'undefined' (so we just don't pass anything in a query) or 'Date'